src/SprykerEco/Service/CrefoPay/Dependency/Service/CrefoPayToUtilTextServiceBridge.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Service\CrefoPay\Dependency\Service;
+
+class CrefoPayToUtilTextServiceBridge implements CrefoPayToUtilTextServiceInterface
+{
+    /**
+     * @var \Spryker\Service\UtilText\UtilTextServiceInterface
+     */
+    protected $utilTextService;
+
+    /**
+     * @param \Spryker\Service\UtilText\UtilTextServiceInterface $utilTextService
+     */
+    public function __construct($utilTextService)
+    {
+        $this->utilTextService = $utilTextService;
+    }
+
+    /**
+     * @param int $length
+     *
+     * @return string
+     */
+    public function generateRandomString(int $length): string
```

Comment:
```suggestion
    public function generateRandomString($length)
```
The original method don't have type hints, in bridges we need to have the same method's definition as in the original class.
You can use the command `console dev:bridge:create ...` for generating correct bridges.